README.md

```diff
@@ -108,6 +108,20 @@ const App = () => {
             // ...
             // errorTransformer={errorTransformer}
             // ...
+
+
+            // logger is an object describing how to log info, warn and errors.
+            // NOTE: logger is not required. If not provided, the logs will be printed to the console.
+            // Example:
+            // const logger = {
```

Comment:
Can we list all the functions that the logger can get in the example? (error, info, warn)
I think it will make the descopers life easier